File: Lista2/C++/Kolo.cpp
#include <iostream>
#include <cmath>
#include "Kolo.h"

Kolo::Kolo(double promien)
{
    this->promien = promien;
}

double Kolo::obwod()
{
    return (2 * 3.14 * promien);
}

double Kolo::pole()
{
    return (3.14 * pow(promien, 2));
}

Kolo::~Kolo()
{
    
}

File: Lista2/Java/Kolo.java
public class Kolo extends Figura
{
    private double promien;

    Kolo(double promien)
    {
        this.promien = promien;
    }

    public double obwod()
    {
        return (2 * 3.14 * promien);
    }

    public double pole()
    {
        return (3.14 * Math.pow(promien, 2));
    }
}